src/SFA.DAS.AssessorService.Application/Mapping/CustomResolvers/ApplicationTypeResolver.cs

```diff
@@ -11,13 +11,26 @@ public class ApplicationTypeResolver : IValueResolver<ApplySummary, ApplicationR
     {
         public string Resolve(ApplySummary source, ApplicationResponse destination, string destMember, ResolutionContext context)
         {
-            if (IsSequenceRequired(source.ApplyData, ApplyConst.ORGANISATION_SEQUENCE_NO) && IsSequenceRequired(source.ApplyData, ApplyConst.STANDARD_SEQUENCE_NO))
+            bool OrganisationSectionRequired(int sectionNo) => 
```

Comment:
I think that this logic needs to be covered by some unit tests

Reply:
Done